login-workflow/docs-new/existing-project-integration.md

```diff
@@ -0,0 +1,217 @@
+
+Once you have a project, you can install this package via:
+
+```shell
+npm install --save @brightlayer-ui/react-auth-workflow
+// or
+yarn add @brightlayer-ui/react-auth-workflow
+```
+
+This package also has a number of peer dependency requirements that you will need to install in your project. To install the latest version of all of these peer dependencies, run the following command in your project root:
+
+```
+npm install --save @mui/material @emotion/react @emotion/styled @brightlayer-ui/colors @brightlayer-ui/react-components date-fns i18next react-dom react-i18next react-router-dom
+// or
+yarn add @mui/material @emotion/react @emotion/styled @brightlayer-ui/colors @brightlayer-ui/react-components date-fns i18next react-dom react-i18next react-router-dom
+```
+
+### Implement AuthUIActions and RegistrationUIActions
+
+You need to implement the backend networking for all networking within react-auth-workflow. Your implementation will likely involve writing calls to your APIs and caching the returned data, as needed, depending on the requirements of your application. The example application has these actions mocked with calls to `sleep` .
+
+1. Create a `/src` folder in your application if it does not already exist
+2. Add an `/actions` folder inside the `src` directory.
+3. Create two files in the new `actions` directory: `AuthUIActions.tsx` and `RegistrationUIActions.tsx`
+    - The first file you created, `AuthUIActions.tsx`, will handle the implementation of the authentication related actions (such as login and forgot password).
+    - The second file you created, `RegistrationActions.tsx`, will handle the implementation of the registration related actions (such as loading the EULA and registration by invitation).
+    - You can copy these files directly from the [example](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/example) project as a starting point and then update the implementation details if you choose.
+4. You might also want to copy over the `example/src/store` and `example/src/constants` folders from react-auth-workflow for the purposes of compiling with the mock `AuthUIActions` and `RegistrationUIActions` before you write your own implementation. These sample implementations make use of the browser LocalStorage, but you may want to use a different approach in order to follow best practices for cybersecurity.
+
+### Authentication
+
+To set up Authentication in your app you will need to import the `AuthContextProvider` and pass in the necessary props. You will pass in your routes as children to the `AuthContextProvider`. More information about React Auth Workflow's exported objects and functions can found in the [API](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/API.md) documentation.
+
+#### Example Usage
+
+Here is an example of how you would set this up using our recommended routing solution, [React Router](https://reactrouter.com/):
+
+```tsx
```

Comment:
General comment: the code examples seem huge. Do yourself (and adopters) a favor and only show the smallest possible example that communicates the point...we have an entire demo project for reference, you don't need to duplicate it in the docs. Don't be afraid to hardcode things in the examples.